app.py

```diff
@@ -338,101 +238,96 @@ def create_tab_content(tab_value):
 
 @app.callback(
     Output("header","children"),
-    [Input(tab_ids['tab_1']['dropdown'],"value")]
+    [Input("cases_dropdown","value")]
 )
 def update_header(value):
-    global df
 
     if value is None or len(value)==0:
-        return "None Selected"
+        return dash.no_update
     else:
         country = value[0]
         return \
             locale.format_string(
                 "As of %s, there have been %d cases in total of COVID-19 confirmed in %s",
-                (df[country].dropna().index[-1].strftime("%d %B %Y"),
-                 df[country].dropna().iloc[-1],
+                (cases[country].dropna().index[-1].strftime("%d %B %Y"),
+                 cases[country].dropna().iloc[-1],
                  country),
                 grouping=True)
```

Comment:
So this bit changes the header to tell you the number of cases in the earliest alphabetical country that you've selected. This seems only very helpful if you have one country selected - do we want to sum them or something if you've selected multiple? that might not be helpful either.
Also this appears to cause the page to reload its webpage title, even though the title always remains as "COVID-19 Cases NZ" anyway...

Reply:
This code is for updating the H3 element at the top of the page. The value of "country' will actually be the first selected item in the dropdown. They're not sorted alphabetically. I put that header in because I wanted a way to quickly see the current total, but I'm not sure if it should be left in its current form, especially if the behavior isn't obvious